README.md

```diff
@@ -35,6 +37,24 @@ None
           s3fs_fuse_access_key_id: <your aws access key id>
           s3fs_fuse_secret_access_key: <your aws secret access key>
 
+### Use EC2 instance profile auth and custom options
+
+```
+- hosts: "tag_class_fuse"
```

Comment:
Does that get used?

Reply:
yes, you can filter hosts by ec2 tags, as long as the tags are assigned upfront. Or is this confusing?

Reply:
It makes sense now. Think it's fine. 👍

Reply:
I've removed it, might be more confusing than helpful as it's not mentioned anywhere else